src/test/java/uk/gov/hmcts/cmc/claimstore/services/DefendantResponseServiceTest.java

```diff
@@ -68,11 +70,28 @@ public void saveShouldFinishSuccessfully() {
         when(claimService.getClaimById(eq(CLAIM_ID))).thenReturn(claim);
 
         //when
-        responseService.save(CLAIM_ID, DEFENDANT_ID, app, AUTHORISATION);
+        responseService.save(CLAIM_ID, DEFENDANT_ID, VALID_APP, AUTHORISATION);
 
         //then
         verify(eventProducer, once())
-            .createDefendantResponseEvent(eq(claim) );
+            .createDefendantResponseEvent(eq(claim));
+    }
+
+    @Test(expected = ForbiddenActionException.class)
```

Comment:
Would be better to subclass from `ForbiddenActionException` so then we can check specific exception. If method throws the same exception for many reasons that kind of test does not quite prove the right exception is thrown. Alternatively asserting error message would be some sort of solution.

Reply:
ok